### ISSUE TITLE
'Please sign in to continue' loop

### ISSUE THREAD

**Issue by erctechdev** [OfficeDev/microsoft-teams-apps-company-communicator#1021]
Everything appears to be set up, however when trying to sign in to Company Communicator Author app, I try and sign in and it just loops back to the sign in screen:

<img width="452" alt="Company Communicator - sign in" src="https://user-images.githubusercontent.com/131354948/233332143-163d99a0-ce74-4624-ba2d-3e3415f4810e.png">

Feedback on previous posters who have experienced this issue have said the **Application ID URI** should be in the Author App Registration. However in the 'official' setup guide, it says the App id URI should be added to the Graph App registration:

<img width="513" alt="Company Communicator - install guide" src="https://user-images.githubusercontent.com/131354948/233333670-54fb7271-4508-4406-bf2c-cf1a57867e4b.png">

To further confuse things, in the Microsoft video, the developer works with the Author App, adding it to this app registration:

https://www.youtube.com/watch?v=uyEpzu53TUg

It is not particularly clear then as to what App registration should actually be configured? 

Can someone help clarify where the setting should be applied and also why it is not able to login? 
Much thanks,
AR

**Comment by gsv022** [comment]
Hi @erctechdev ,
Thank you for providing the query.

- To clarify, the main branch in the CC repository currently contains the latest code, which is version 5.3. This version requires three app registrations, namely author, user, and graph. The [deployment guide](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/wiki/Deployment-guide) for version 5.3 provides instructions for setting up the authentication process for the graph app registration.

- On the other hand, version 4.x (specifically, version 4.1.5) only requires two app registrations, author and user. The [deployment guide](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/blob/v4.x/Wiki/Deployment-guide.md) for version 4.x provides instructions for setting up the authentication process for the author app registration.

- Regarding  the Microsoft video that was referred to, it shows the deployment procedure for version 4.x.

- In summary, if you are deploying version **4.x**, you need to set up the authentication process for the **author app** registration. If you are deploying the latest version, which is version **5.3**, you need to set up the authentication process for **graph app** registration

I hope this information clarifies the matter. Please let me know if you have any further questions.

**Comment by erctechdev** [comment]
Clears it all up, thank you...now to try and get it working!

**Comment by erctechdev** [comment]
Using Chrome developer console, I can see this message in red:

**Login succeeded!
Error from getAuthToken:  App resource defined in manifest and iframe origin do not match**

After pouring over the config settings it all looks in order, so now sure why it isn't working. Perhaps I might need to delete everything and re-install, because I'm out of ideas.
Thanks
AR

**Comment by gsv022** [comment]
@erctechdev , its good that you are going for new deployment. As it appears that your query has been successfully addressed, we will mark this issue request as closed. Thank you for your cooperation. Kindly be informed that in the event of any obstacles encountered during the process by logging a new issue.
